### DISCUSSION TITLE
v2022.6.13-pre

### DISCUSSION THREAD

**Discussion by CrazyBoyFeng** [CrazyBoyFeng/AccSettings#10] [Announcements]
[ACC v2022.6.12-dev](https://github.com/VR-25/acc/releases/tag/v2022.6.12-dev) inside

**Full Changelog**: https://github.com/CrazyBoyFeng/AccSettings/compare/v2022.6.7...v2022.6.13-pre

<hr /><em>This discussion was created from the release <a href='https://github.com/CrazyBoyFeng/AccSettings/releases/tag/v2022.6.13-pre'>v2022.6.13-pre</a>.</em>

**Comment by anuraag488** [comment]
@CrazyBoyFeng new update please.
